setup.cfg

```diff
@@ -125,17 +126,22 @@ exclude =
     node_modules,
     */migrations/*
 
-[pydocstyle]
-convention = google
-match_dir = ^(?!tests|venv|build|scripts).*
-match = ^(?!(setup)\.(py)$).*\.(py)$
-
 [isort]
 known_ralph=ralph
 sections=FUTURE,STDLIB,THIRDPARTY,RALPH,FIRSTPARTY,LOCALFOLDER
 skip_glob=venv
 profile=black
 
+[pydocstyle]
+convention = google
+match_dir = ^(?!tests|venv|build|scripts).*
+match = ^(?!(setup)\.(py)$).*\.(py)$
+
+[mypy]
+warn_return_any = True
```

Comment:
Would a [disallow_untyped_defs = True](https://mypy.readthedocs.io/en/stable/config_file.html#confval-disallow_untyped_defs) be too much?